### PR TITLE
Translator-onboarding guide - Issue #7115

### PR DESCRIPTION
**Pull request by banjo-tech**
_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: docs/contributing/translator_onboarding.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:1]:
```suggestion
# Translations
```

No need to do all caps here.

File: docs/contributing/translator_onboarding.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:3]: Just remove any added whitespace on these and other lines.

```suggestion
Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
```
`Transifex` is a proper noun so should always have a capital T

File: docs/contributing/translator_onboarding.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:5]:
```suggestion
## Translation workflow
```

File: docs/contributing/translator_onboarding.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
+   Each new release has about two weeks release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated when there is a release candidate until the final release. If translations reach a threshold of X% ( this is about 90) we communicate that this is an officially supported translation in the docs.
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:6]:
```suggestion
Translations from Transifex are only integrated into the Wagtail repository at the time of a new release. When a release is close to being ready there will be a RC (Release Candidate) for the upcoming version and the translations will be exported to Transifex.

During this RC period, usually around two weeks, there will be a chance for all the translators to update and add new translations. We will also notify the `#translators` channel in the Watail slack group at this time.

These new translations are imported into Wagtail for any subsequent RC and the final release.

If translations reach a threshold of about 90% we communicate that this is an officially supported translation in the docs.
```

Have a read, and see what you think.

File: docs/contributing/translator_onboarding.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
+   Each new release has about two weeks release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated when there is a release candidate until the final release. If translations reach a threshold of X% ( this is about 90) we communicate that this is an officially supported translation in the docs.
+
+
+## What is transifex
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:9]:
```suggestion
## What is Transifex
```

File: docs/contributing/translator_onboarding.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+ Wagtail is localized (translated) using [transifex](https://www.transifex.com/). A software and web localization platform that allows organisations to coordinate translation projects and collaborate with communities to translate materials. 
+
+
+## To help out in translating wagtail here's what you will do
+
+- Introduce yourself on [slack]( https://wagtailcms.slack.com/archives/CTKN6UXKN) to join  fellow translators on slack.
+- Request access to become a member of the language team you want to work with on slack. Mention your transifex username when you make this request.
+-	Go to [transifex](https://www.transifex.com/).
```
Comment (lb-) [inline docs/contributing/translator_onboarding.md:18]: put all these as a flat list with the same indentation.

Also, no need for a period (full stop) at the end of each list item.

**Review by lb- — CHANGES_REQUESTED** [review]
@banjo-tech  great start - I have put some recommendations and have some high level feedback below.

1. Remember to include the issue you are trying to fix in the PR's description
2. Please check your changes, it looks like you have some how deleted / re-written a few files.
3. Let's rename `docs/contributing/translator_onboarding.md` to `docs/contributing/translations.md` I think.

@loicteixeira would love your feedback also - I think we want to just start with a basic overview to resolve https://github.com/wagtail/wagtail/issues/7115 and then can build additional content under new issues.

**Comment by banjo-tech** [comment]
Thank you for the feedback @lb- I would make the necessary corrections and update the pr

File: docs/contributing/translator_onboarding.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
+   Each new release has about two weeks release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated when there is a release candidate until the final release. If translations reach a threshold of X% ( this is about 90) we communicate that this is an officially supported translation in the docs.
+
+
+## What is transifex
+
+ Wagtail is localized (translated) using [transifex](https://www.transifex.com/). A software and web localization platform that allows organisations to coordinate translation projects and collaborate with communities to translate materials.
```
Comment (tomdyson) [inline docs/contributing/translator_onboarding.md:11]:
```suggestion
 Wagtail is localised (translated) using [Transifex](https://www.transifex.com/), a web platform that helps organisations coordinate translation projects.
```

File: docs/contributing/translator_onboarding.md (hunk 4 above)
Comment (lb-) [reply docs/contributing/translator_onboarding.md:6]: Sorry. It's actually 90% as first suggested. 

As per https://github.com/wagtail/wagtail/wiki/Managing-Wagtail-translations#fetching-new-translations-from-transifex

File: docs/contributing/translator_onboarding.md (hunk 6 above)
Comment (banjo-tech) [reply docs/contributing/translator_onboarding.md:24]: @lb- I am not familiar with what a flat list is can you please explain?

**Comment by banjo-tech** [comment]
Hi @lb, I just created a new PR for the update requested for.

https://github.com/wagtail/wagtail/pull/9431

**Comment by lb-** [comment]
If possible. Can you just keep the one PR when making changes? It's hard to follow all the conversations and feedback across multiple PRs. 

You can just push new changes to the existing branch and the PR will automatically be updated.

**Comment by banjo-tech** [comment]
Definitely. 
I created a new PR because of the merge conflict.
